docs/support_playbook.md

```diff
@@ -120,8 +120,19 @@ copier = Courses::CopyToProviderService.new(sites_copy_to_course: Sites::CopyToC
 # Handles edge case where .published returns withdrawn
 provider.courses.published.filter { |c| c.content_status != :withdrawn }.each do |course|
   new_course = copier.execute(course:, new_provider: provider_to_copy_to)
-  
+
   # Set the accredited body if needed
   new_course.update(accredited_body_code: "2N2")
 end
 ```
+
+## Visa sponsorship issue
+
+If providers encounter an error message 'Select if visas can be sponsored' when publishing a course after correctly filling the visa option on basic details page, they need to have the provider sponsorship attributes updated from nil, as we are validating on the provider rather than the course at present.
+
+```ruby
+# Find provider to update
+provider = RecruitmentCycle.current.providers.find_by(provider_code: "1TZ")
+
+provider.update(can_sponsor_skilled_worker_visa: true, can_sponsor_student_visa: true)
```

Comment:
Does it matter whether we set these to `true` or `false`?

Reply:
When creating courses, depending on the provider type, these values (at the course level) will be defaulted to what the provider has set. (eg a scitt creating a course could have these defaulted to what the accrediting body has set even though it can be changed)

Reply:
No, the validation is ```def validate_provider_visa_sponsorship_publishable
    if provider.can_sponsor_student_visa.nil? || provider.can_sponsor_skilled_worker_visa.nil?
      errors.add(:base, :visa_sponsorship_not_publishable)
    end
  end```

Reply:
but is seems more logical?